source/testing/index.md

```diff
@@ -1,79 +1,149 @@
 Testing is a core part of the Ember framework and its development cycle.
 
 Let's assume you are writing an Ember application which will serve as a blog.
-This application would likely include models such as `user` and `post`. It would
-also include interactions such as _login_ and _create post_. Let's finally
-assume that you would like to have [automated tests] in place for your application.
+This application would likely include models such as `user` and `post`.
+It would also include interactions such as _login_ and _create post_.
+Let's finally assume that you would like to have [automated tests] in place for your application.
 
-There are three different classifications of tests that you will need:
-**Acceptance**, **Unit**, and **Integration**.
+There are four different types of test setups you can choose from depending on what you will need to test.
+Let's have a look at those test types and when each of them is the most useful.
 
-### Acceptance Tests
+### Application Tests
 
-Acceptance tests are used to test user interaction and application flow.  The tests interact
-with the application in the same ways that a user would, by doing things like filling out
-form fields and clicking buttons.  Acceptance tests ensure that the features within
-a project are basically functional, and are valuable in ensuring the core features of a
-project have not regressed, and that the project's goals are being met.
+Some tests will require you to test user interaction and application flow.
+In these kind of tests you would like to be able to interact with the application in the same ways that a user would,
+by doing things like filling out form fields and clicking buttons.
```

Comment:
In these kind of tests, you interact with the application in the same ways that a user would, such as filling out form fields and clicking buttons.